libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -30,6 +30,7 @@
 #include <AP_Notify/AP_Notify.h>
 #include <AP_Vehicle/AP_Vehicle_Type.h>
 #include <GCS_MAVLink/GCS.h>
+#include <AP_NMEA_Output/AP_NMEA_Output.h>
```

Comment:
It is already mentioned libraries/AP_AHRS/AP_AHRS.h

Reply:
Fixed.